src/main/java/com/google/cloud/genomics/dataflow/pipelines/VariantSimilarity.java

```diff
@@ -46,21 +55,29 @@
  */
 public class VariantSimilarity {
   private static final String VARIANT_FIELDS
-      = "nextPageToken,variants(start,calls(genotype,callSetName))";
+    = "nextPageToken,variants(start,calls(genotype,callSetName))";
+
+  public static interface VariantSimilarityOptions extends GenomicsDatasetOptions, GCSOptions {
+    @Default.Boolean(true)
+    Boolean getUseAlpn();
```

Comment:
For context about that option, see https://github.com/googlegenomics/start-here/pull/51/files#r32270199

I will add the comment in both places where we have that.

Thanks!